ui/admin/app/controllers/scopes/scope/app-tokens/new.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Controller from '@ember/controller';
+import { service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { restartableTask } from 'ember-concurrency';
+import { GRANT_SCOPE_CHILDREN } from 'api/models/role';
+import { TYPE_SCOPE_PROJECT } from 'api/models/scope';
+class FilterOptions {
```

Comment:
FYI in `main` we have a separate class for [this](https://github.com/hashicorp/boundary-ui/blob/013aa2e5706bc9ec7e44f0fd77bf3a591e39c34b/ui/admin/app/utils/filter-options.js)